src/main/java/com/mineinabyss/looty/ecs/action/IncreaseDurabilityAction.kt

```diff
@@ -0,0 +1,17 @@
+package com.mineinabyss.looty.ecs.action
+
+import com.mineinabyss.geary.ecs.api.actions.GearyAction
+import com.mineinabyss.geary.ecs.api.entities.GearyEntity
+import com.mineinabyss.looty.ecs.components.DurabilityComponent
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("looty:increaseDurability")
```

Comment:
Use underscores instead of camelCase for serial names. I've not given it much thought but it's what bedrock edition did and I wanted to mimic its naming scheme for some components.